src/panels/config/energy/components/ha-energy-grid-settings.ts

```diff
@@ -220,6 +220,7 @@ export class EnergyGridSettings extends LitElement {
           ${this._co2ConfigEntry
             ? html`<div class="row" .entry=${this._co2ConfigEntry}>
                 <img
+                  alt=""
```

Comment:
There text accompanying this image, therefore I believe the image to be decorative.